internal/builders/bazel/README.md

```diff
@@ -0,0 +1,253 @@
+  - [Referencing the Bazel builder](#referencing-the-bazel-builder)
+  - [Private Repositories](#private-repositories)
+  - [Supported Triggers](#supported-triggers)
+  - [Workflow Inputs](#workflow-inputs)
+  - [Workflow Outputs](#workflow-outputs)
+  - [Provenance Format](#provenance-format)
+  - [Provenance Example](#provenance-example)
+
+<!-- tocstop -->
+
+---
+
+## Benefits of Provenance
+
+Using the Bazel builder will generate a non-forgeable attestation to the
+Bazel build artifacts produced using the identity of the GitHub workflow. This can be used to
+create a positive attestation to an artifact coming from your repository.
+
+That means that once your users verify the artifact they have downloaded they can
+be sure that it was created by your repository's workflow and hasn't been
+tampered with.
+
+## Development status
+
+The Bazel builder is currently in development.
+
+Please try it out and
+[create an issue](https://github.com/slsa-framework/slsa-github-generator/issues/new)
+to send us feedback!
+
+## Generating Provenance
+
+The Bazel builder uses a Github Actions reusable workflow to build your
+artifact(s) and generate the provenance.
+
+### Getting Started
+
+Let's assume you have a Bazel project which includes a
+[WORKSPACE file](https://bazel.build/reference/glossary#workspace-file)
+at the root of your repository as well as one or more
+[BUILD files](https://bazel.build/reference/glossary#build-file). Also included
+at the root of the directory is the
+[.bazelversion file](https://github.com/bazelbuild/bazelisk#how-does-bazelisk-know-which-bazel-version-to-run)
+, [.bazelrc file](https://bazel.build/run/bazelrc)
+, and a
+[MODULE.bazel file](https://bazel.build/rules/lib/globals/module).
+
+To get started, you will need to add some steps to your current workflow. We
+will assume you have an existing Github Actions workflow to build your project.
+This assumes that the files above are in the root directory of your
+repository.
+
+We also define the targets to be built as 'fib' and 'hello', and
+we define the flag of the build to be '--strip=always'
+
+The following reusable workflow call will build the targets and upload the artifacts into a .zip and
+generate provenance attestations which will be uploaded as another .zip to the
+workflow run.
+
+```yaml
+jobs:
+  build:
+    permissions:
+      id-token: write # For signing
+      contents: read # For repo checkout.
+      actions: read # For getting workflow run info.
+    if: startsWith(github.ref, 'refs/tags/')
+    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_bazel_slsa3.yml@v1.7.0
+    with:
+      targets: "//src:fib //src:hello"
+      flags: "--strip=always"
+```
+
+The `targets` are a set of space separated build targets to be built. The targets are assume to
+be referenced from the root of the directory. Because of this the each target inputted should be referenced
+as `//path/from/root/to/target:your_target`.
```

Comment:
We should also mention that the target identifier needs to be specified. Bazel allows you to specify just a package name and defaults to a target id with the same name as the package name (e.g. `bazel build //test` will build `//test:test`)
```suggestion
The `targets` are a set of space separated build targets to be built. Each target must include the `//` workspace root identifier and package target identifier (`:your_target`). Because of this each target should be of the form `//path/from/root/to/target:your_target`.
```

Reply:
Thank you for the clarification!